src/components/SignalCard/SignalCard.js

```diff
@@ -4,9 +4,12 @@ import cx from 'classnames'
 import Panel from '@santiment-network/ui/Panel/Panel'
 import Icon from '@santiment-network/ui/Icon'
 import Toggle from '@santiment-network/ui/Toggle'
+import Button from '@santiment-network/ui/Button'
+import { DesktopOnly, MobileOnly } from './../../components/Responsive'
 import MultilineText from '../../components/MultilineText/MultilineText'
 import StatusLabel from './../../components/StatusLabel'
 import { RemoveSignalButton, SignalTypeIcon } from './controls/SignalControls'
+import { ContextMenu } from '@santiment-network/ui'
```

Comment:
This import statement [does not match our style guide](https://github.com/santiment/san-style-guide#webpack-tree-shaking-problem)

Reply:
Fixed